package.json

```diff
@@ -7,6 +7,13 @@
   "bin": {
     "pino": "./pretty.js"
   },
+  "files" : [
+    "pino.js", 
+    "browser.js",
+    "pretty.js",
+    "LICENEE",
```

Comment:
I adhered.
let me know if you change your mind again, I'll be happy to change it.